diff --git a/sshfs.c b/sshfs.c
index 2dcc2f6..9c09513 100644
--- a/sshfs.c
+++ b/sshfs.c
@@ -3766,6 +3766,9 @@ static int sshfs_opt_proc(void *data, const char *arg, int key,
 			 * Furthermore the mountpoint must NOT exist prior to mounting.
 			 * So we cannot use realpath(3).
 			 */
+			if ('\0' == arg[0] || ('*' == arg[0] && '\0' == arg[1]))
+				sshfs.mountpoint = strdup(arg);
+			else
 			if ((('A' <= arg[0] && arg[0] <= 'Z') || ('a' <= arg[0] && arg[0] <= 'z'))
 				&& ':' == arg[1] && '\0' == arg[2]) {
 				/* drive: make a copy */
